altair/vegalite/v2/examples/line_percent.py

```diff
@@ -0,0 +1,23 @@
+"""
+Line Chart with Percent axis
+-----------------------
+This example shows how to set an axis as a percent.
+"""
+
+import altair as alt
+from vega_datasets import data
+
+source = data.jobs()
+
+source.year = source.year.astype(str)
```

Comment:
Could you add a comment about why we convert the column to a string? Since this will be an example in the documentation, it will help to err toward didacticism.